src/api/contracts.ts

```diff
@@ -0,0 +1,38 @@
+import TruffleContract from 'truffle-contract'
+
+const contractNames = [
+  'DutchExchange',
+  'DutchExchangeFactory',
+  'DutchExchangeETHGNO',
+  'DutchExchangeGNOETH',
+  'Token',
+  'TokenETH',
+  'TokenGNO',
+]
+
+const Contracts = contractNames.map(name => TruffleContract(require(`../../build/contracts/${name}.json`)))
```

Comment:
Could you rename to `ContractJSON` or `ContractArtifacts` ? 😅

Reply:
These are not contract artifacts, they are contract classes, if you will. Since each is `TruffleContract(artifact)`

Reply:
oh I didn't see the TruffleContract before, sneaky. Nvm then